### PR TITLE
Fix editfeatureselector example

### PR DESCRIPTION
**Pull request by llienher**
Stéphane converted the php-date-formatter lib into a webpack custom supported version we can use find in the src/misc directory. Other changes are minor syntax errors in webpack configs and import of css.

### CODE REVIEW

File: buildtools/webpack.commons.js [hunk 1]
```diff
@@ -91,13 +91,13 @@ const typeaheadRule = {
...
-const dateFormatterRule = {
-  test: require.resolve('jquery-datetimepicker'),
+/*const dateFormatterRule = {
+  test: require.resolve('../src/misc/php-date-formatter.js'),
   use: {
     loader: 'expose-loader',
     options: 'DateFormatter'
   }
-};
+};*/
```
Comment (llienher) [inline buildtools/webpack.commons.js]: @sbrunner Do you prefer we remove these lines or we keep it what a additional informative comment ?

Comment (sbrunner) [reply buildtools/webpack.commons.js]: +1 to remove

File: buildtools/webpack.commons.js [hunk 2]
```diff
@@ -165,7 +165,7 @@ const config = {
       olcsRule,
       angularRule,
       typeaheadRule,
-      dateFormatterRule,
+//      dateFormatterRule,
```
Comment (sbrunner) [inline buildtools/webpack.commons.js]: same

File: buildtools/webpack.commons.js [hunk 3]
```diff
@@ -198,8 +198,9 @@ const config = {
       'jsts': 'jsts/org/locationtech/jts',
       'ol/ol.css': 'openlayers/css/ol.css',
       'ol': 'openlayers/src/ol',
-      'jquery-ui/datepicker' : 'jquery-ui/ui/widgets/datepicker', // For angular-ui-date
+      'jquery-ui/datepicker': 'jquery-ui/ui/widgets/datepicker', // For angular-ui-date
       'proj4': 'proj4/lib',
+      'jquery-datetimepicker$': 'jquery-datetimepicker/jquery.datetimepicker.js',
```
Comment (sbrunner) [inline buildtools/webpack.commons.js]: extra $

File: src/misc/php-date-formatter.js [hunk 4]
```diff
@@ -0,0 +1,612 @@
...
+        }
+        vDateStr += str;
+      }
+      return vDateStr;
+    }
+    return '';
+  }
+};
+
+export default DateFormatter;
```
Comment (sbrunner) [inline src/misc/php-date-formatter.js]: missing endline

Comment (llienher) [reply src/misc/php-date-formatter.js]: Done

**Pull request merged**